### PR TITLE
Dynamic Grid Filters, Updated Foundatio and OTEL

### PR DESCRIPTION
**Pull request by niemyjski**
image
image

- [x] Need to refactor to a builder type pattern so child controls register themselves with the parent builder component and render automatically.
- [x] This will support Organization filtering
- [x] This is just simple filtering; I think it's a good idea to only have 1 per type of filter.
- [x] User can also type a filter in via ad hoc textbox filter into part of this as well.

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -121,17 +121,10 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Setup .NET Core
-        uses: actions/setup-dotnet@v4
-        with:
-          dotnet-version: 8.0.*
-          #dotnet-quality: preview
-
```
Comment (ejsmith) [inline .github/workflows/build.yaml:129]: Why did you remove this?

Comment (niemyjski) [reply .github/workflows/build.yaml:129]: This is for the client build. There is no reason to install .NET for it.
